src/Microsoft.Health.Dicom.SqlServer/Microsoft.Health.Dicom.SqlServer.csproj

```diff
@@ -72,6 +72,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Remove="Features\Schema\Migrations\7.diff.sql" />
```

Comment:
What are these None remove tags helping with?

Reply:
This is not needed, can be removed. Added by vs automatically